qt/scientific_interfaces/Inelastic/Manipulation/InelasticDataManipulationElwinTab.h

```diff
@@ -84,7 +79,8 @@ class MANTIDQT_INELASTIC_DLL InelasticDataManipulationElwinTab : public Inelasti
   std::string getOutputBasename();
   MatrixWorkspace_sptr getInputWorkspace() const;
   MatrixWorkspace_sptr getPreviewPlotWorkspace();
-  void checkForELTWorkspace();
+  bool checkForELTWorkspace();
+  std::string prepareOutputPrefix(std::vector<std::string> workspaceNames);
```

Comment:
Can these functions be const also? And can the parameter to `prepareOutputPrefix` be `std::vector<std::string> const &`?

Reply:
``prepareOutput`` was giving me problems as it is called with a const function input. I have tried a different way, I hope the test don't complain. 
``checkForEltWorkspace()`` can't be made ``const`` if calling ``getOutputBasename`` from within.